### PR TITLE
Allow omitting v-prefix

### PR DESCRIPTION
**Pull request by sascha-andres**
Add option to not prefix ther generated versions with a literal v.

---
This change is [Reviewable](https://reviewable.io/reviews/pantheon-systems/autotag/16)

### CODE REVIEW

File: autotag/main.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ type Options struct {
 	RepoPath            string `short:"r" long:"repo" description:"Path to the repo" default:"./" `
 	PreReleaseName      string `short:"p" long:"pre-release-name" description:"create a pre-release tag with this name (can be: alpha|beta|pre|rc)"`
 	PreReleaseTimestamp string `short:"T" long:"pre-release-timestamp" description:"create a pre-release tag and append a timestamp (can be: datetime|epoch)"`
+	NoVersionPrefix     bool   `short:"-e" long:"empty-version-prefix" description:"do not prepend v to version tag"`
```
Comment (netflash) [inline autotag/main.go]: All of the above short parameters don't have a `-` dash sign. Not sure if you want it here.

Comment (sascha-andres) [reply autotag/main.go]: Removed. Not sure why I added it in the first place

**Comment by sascha-andres** [comment]
@netflash any news on this pr?

**Comment by spheromak** [comment]
@sascha-andres so sorry for not seeing this earlier. Better late than never tho. This LGTM and I will merge in the next day or 2 after getting some of the CI updated on the repo.

**Comment by sascha-andres** [comment]
@spheromak any news on this?

**Comment by spheromak** [comment]
@sascha-andres sorry for taking so long. This fell back onto my radar, and I hope to get it done this week.

**Comment by sascha-andres** [comment]
@spheromak any eta?

**Comment by ghoussard** [comment]
This feature is very interresting. It is possible to merge this PR please ?

**Comment by joemiller** [comment]
Due to recent commits containing major refactoring of the tests this PR had large amounts of conflicts. I took the main body of code and ported it over to a new branch with new tests. #34 

Thank you for the submission @sascha-andres !

**Pull request closed without merging**